projects/ngx-stories/src/lib/ngx-stories.component.html

```diff
@@ -39,5 +39,14 @@ <h5 [style.visibility]="isHolding ? 'hidden' : 'visible'">
   <button class="prev" (click)="navigateStory('previous')">Previous</button>
   }
   <button class="next" (click)="navigateStory('next')">Next</button>
+
+
+  <!-- Next button -->
+  <div class="desktop-navigation next-btn" role="button" tabindex="0" (click)="navigateStory('next')"></div>
```

Comment:
Any particular reason that we are not using `<button>` tag here ?
Just want to know.

Reply:
My bad. We can have button there